tests/framework/installer/ceph_installer.go

```diff
@@ -396,20 +396,20 @@ func (h *CephInstaller) InstallRook(namespace, storeType string, usePVC bool, st
 	logger.Infof("Installing rook on k8s %s", k8sversion)
 
 	startDiscovery := true
-	onamespace := namespace
+	onamespace := operatorNamespace
 	// Create rook operator
 	if h.useHelm {
 		// disable the discovery daemonset with the helm chart
 		settings := "enableDiscoveryDaemon=false"
 		startDiscovery = false
-		err = h.CreateRookOperatorViaHelm(namespace, settings)
+		err = h.CreateRookOperatorViaHelm(onamespace, settings)
 		if err != nil {
 			logger.Errorf("Rook Operator not installed ,error -> %v", err)
 			return false, err
 
 		}
 	} else {
-		onamespace = SystemNamespace(namespace)
+		onamespace = SystemNamespace(onamespace)
```

Comment:
It would be better if `SystemNamespace()` were called when the test is first initialized. Then we could cleanup the call to `SystemNamespace()` in other places throughout the tests. I'll add an example below.